ONSdigital/eq-author-app
# Add scrollToTop only on selected scrollPanes

### What is the context of this PR?

This PR allows us to choose which scroll panes reset when changing pages

### How to review 

1. Crank out a questionnaire
2. Start duplicating a bunch of questions
3. Duplicate a bunch of some sections
4. Scroll down to the bottom of the questionnaire panel and the left hand nav
5. Switch page
6. Only the questionnaire panel and properties panel should reset 

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
